recipe/meta.yaml

```
@@ -1,38 +1,45 @@
-{% set version = "1.9.11" %}
+{% set version = "1.9.16" %}
 
 package:
   name: bwidget
   version: {{ version }}
 
 source:
   url: https://downloads.sourceforge.net/project/tcllib/BWidget/{{ version }}/bwidget-{{ version }}.tar.gz
-  sha256: 448a1db08a4ca477ce339dfe592fd85c4c0649c3ff554fb4e86d6826d39bc8b9
+  sha256: bfe0036374b84293d23620a7f6dda86571813d0c7adfed983c1f337e5ce81ae0
+  # use a separate folder to avoid moving scripts generated by conda-build
+  folder: src
 
 build:
-  skip: True  # [win]
-  number: 1
-  script:
-    # Remove this stuff so that lib is not polluted
-    - rm -Rf .fslckout ChangeLog tests demo BWman
-    - "[[ ! -d $PREFIX/lib/tcl8.6/bwidget ]] || exit 1"
-    - "mkdir -p $PREFIX/lib/tcl8.6/bwidget"
-    - "mv * $PREFIX/lib/tcl8.6/bwidget/"
+  number: 0
 
 requirements:
+  host:
+    - tk {{ tk }}
   run:
     - tk
 
 test:
   commands:
     - echo "if {[catch {package require -exact BWidget {{ version }}; exit 0}]} {exit 1}" | xvfb-run tclsh  # [linux]
     - echo "if {[catch {package require -exact BWidget {{ version }}; exit 0}]} {exit 1}" | tclsh  # [osx]
+    - echo package require BWidget;exit | tclsh --  # [win]
```

Shouldn't this be like `tktable` and have the selector `[not linux]`?

```suggestion
    - echo "if {[catch {package require -exact BWidget {{ version }}; exit 0}]} {exit 1}" | tclsh  # [not linux]
```

---

Fixed